tests/integration/kubernetes/filter_k8s_test.sh

```diff
@@ -35,7 +35,19 @@ main()
 		local flag="false"
 		for SKIP_ENTRY in "${_K8S_SKIP_UNION[@]}"
 		do
-			SKIP_ENTRY="${SKIP_ENTRY#- }.bats"
+			# Remove all characaters including and before the matching pattern '- '
+			# Example: " - test.bats" -> "test.bats"
+			SKIP_ENTRY="${SKIP_ENTRY#- }"
```

Comment:
I think this comment is wrong as this only removes a starting `- `, not characters before it?:
```
SKIP_ENTRY=" - test.bats"
echo "${SKIP_ENTRY#- }"
 - test.bats
```
I'm not sure it makes the code wrong as we get the SKIP_ENTRY from yaml, so it shouldn't be mismatching?

Reply:
Yes, you are right. The expansion only removes the matching characters. The string parsed by yq should start with `-`.  I will update the comment. Thanks!